### DISCUSSION TITLE
Integrate api/v2/settings/objects in Monaco

### DISCUSSION THREAD

**Discussion by roger-mario** [Dynatrace/dynatrace-configuration-as-code#716] [Q&A] [answered]
Hi all

I would like to read out configurations from the v2/settings/objects API - for example OS Service Monitoring.
Sadly the given response from the API doesn't match the format for a new API in Monaco, see picture below.

![image](https://user-images.githubusercontent.com/103379642/185363583-70847041-2eeb-45d9-b02e-2c239a0484ba.png)

Docu from monaco for new api's: https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/Guides/add_new_api

How can you integrate these settings?

Best regards,
Roger

**Comment by UnseenWizzard** [comment, marked as answer]
Hi @roger-mario, 
sadly the settings v2 API is quite complex to support and it's currently not possible to use monaco for configuring settings. 

We are working on supporting this in an upcoming release, but I can not give you an ETA at this time.

Best, 
Nico

**Comment by roger-mario** [comment]
Hi @UnseenWizzard 

Thank you for your quick reply. I can understand the issue and am happy to hear that it might be integrated in a upcoming release.

Greetings
